content/en/docs/releases/kubeflow-1.9.md

```diff
@@ -22,7 +22,7 @@ weight = 95
           <a href="https://blog.kubeflow.org/kubeflow-1.9-release/">Kubeflow 1.9 Release Announcement</a>
         <br>
         <b>Video:</b> 
-          To be published
+          <a href="https://www.youtube.com/watch?v=3sZzE-vPsMI">Kubeflow 1.9 Release Update</a>
```

Comment:
Please can you upload this video to the Kubeflow YouTube channel: https://www.youtube.com/kubeflow
@james-jwu and @zijianjoy should help you with that.

Reply:
done with 207197a